{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "ac075f3b_7594cccc",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1064003
      },
      "writtenOn": "2023-10-25T23:27:02Z",
      "side": 1,
      "message": "Looking into boot failure that wasn\u0027t present on internal git_main",
      "revId": "c0a34a3ced664dc7ef34b606eb821fc9331780b6",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "ee70ca5f_70cb564d",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1064003
      },
      "writtenOn": "2023-10-26T19:13:01Z",
      "side": 1,
      "message": "Looks like preexisting b/307755000, but I\u0027ll wait until that\u0027s resolved before landing.",
      "parentUuid": "ac075f3b_7594cccc",
      "revId": "c0a34a3ced664dc7ef34b606eb821fc9331780b6",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}